server/config.go

```diff
@@ -2,7 +2,7 @@ package server
 
 import (
 	"github.com/intervention-engine/fhir/auth"
-	"gopkg.in/mgo.v2"
+	mgo "gopkg.in/mgo.v2"
```

Comment:
What is the purpose of this?  It seems unnecessary.  Probably not worth a commit to back it out, but I'm just wondering if you ran into something that required it?